packages/web-runtime/src/container/versions.ts

```diff
@@ -15,6 +15,5 @@ export const getBackendVersion = ({
   }
   const product = backendStatus.product || 'OpenCloud'
   const version = backendStatus.productversion || backendStatus.versionstring
-  const edition = backendStatus.edition
```

Comment:
`const edition = backendStatus.edition || ''` should already do the trick, right? We want the edition to be shown if it's set. We just don't want a nullish string to be printed. 🙈 The rest of the PR can be reverted, we do want to keep the functionality.